app/controllers/api/extradition_flight_controller.rb

```diff
@@ -0,0 +1,63 @@
+module Api
+  class ExtraditionFlightController < ApiController
+    before_action :set_extradition_flight, only: %i[index update]
+
+    PERMITTED_NEW_PARAMS = [
+      :type,
+      {
+        attributes: %i[flight_number flight_time],
+        relationships: [{ move: {} }],
+      },
+    ].freeze
+
+    PERMITTED_UPDATE_PARAMS = [:type, { attributes: %i[flight_number flight_time] }].freeze
+
+    def index
+      render_extradition_flight(@extradition_flight, :ok)
+    end
+
+    def create
+      @extradition_flight = ExtraditionFlight.new(new_extradtion_flight_attributes)
+
+      @extradition_flight.save!
```

Comment:
```suggestion
      @extradition_flight = ExtraditionFlight.create!(new_extradtion_flight_attributes)
```

I know you didn't write this, but it can be simplified